packages/design-system-tailwind-preset/src/typography.test.ts

```diff
@@ -82,6 +82,20 @@ describe('Typography', () => {
     '--typography-s-body-lg-medium-font-weight',
     '--typography-s-body-lg-medium-letter-spacing',
     '--typography-l-body-lg-medium-font-weight',
+    '--typography-s-display-md-font-weight',
+    '--typography-s-heading-lg-font-weight',
+    '--typography-s-heading-md-font-weight',
+    '--typography-s-heading-sm-font-weight',
+    '--typography-s-body-md-font-weight',
+    '--typography-s-body-sm-font-weight',
+    '--typography-s-body-xs-font-weight',
+    '--typography-l-display-md-font-weight',
+    '--typography-l-heading-lg-font-weight',
+    '--typography-l-heading-md-font-weight',
+    '--typography-l-heading-sm-font-weight',
+    '--typography-l-body-md-font-weight',
+    '--typography-l-body-sm-font-weight',
+    '--typography-l-body-xs-font-weight',
```

Comment:
Should we remove these from the design tokens CSS?

Reply:
They're being used in the stories so I'm keeping them in for now